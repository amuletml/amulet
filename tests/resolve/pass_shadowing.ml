let x = 1

let x = 2

let a = x

let x =
  let x = 4
  x

let b = x
