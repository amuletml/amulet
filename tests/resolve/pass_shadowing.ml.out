let x#1 = 1
let x#2 = 2
let a#3 = x#2
let x#4 = let x#5 = 4
  in x#5
let b#6 = x#4
